packages/suite-native-settings/src/components/DeviceSettings.tsx

```diff
@@ -1,20 +1,37 @@
+import { useNavigation } from '@react-navigation/native';
 import React from 'react';
+import { SettingsScreenProp, SettingsStackRoutes } from '../navigation/routes';
 
 import { SettingsSection } from './SettingsSection';
 import { SettingsSectionItem } from './SettingsSectionItem';
 
-export const DeviceSettings = () => (
-    <SettingsSection title="Device">
-        <SettingsSectionItem title="Customization" iconName="palette" subtitle="Name, Homescreen" />
-        <SettingsSectionItem
-            title="Security"
-            iconName="lock"
-            subtitle="PIN Active, Passphrase enabled"
-        />
-        <SettingsSectionItem
-            title="Danger Area"
-            iconName="lock"
-            subtitle="Factory reset, Custom firmware"
-        />
-    </SettingsSection>
-);
+export const DeviceSettings = () => {
+    const navigation = useNavigation<SettingsScreenProp>();
+
+    const handleNavigate = (route: SettingsStackRoutes): void => {
```

Comment:
seems little bit unecessary, just do `const {navigate} = useNavigation()` and then use directly navigate below

Reply:
[b608371](https://github.com/trezor/trezor-suite/pull/5680/commits/b608371eec683b811768423618096c20c415c35c)